src/fu-engine.c

```diff
@@ -2423,6 +2423,15 @@ fu_engine_schedule_update(FuEngine *self,
 	g_autoptr(FuHistory) history = NULL;
 	g_autoptr(GFile) file = NULL;
 
+#ifndef HAVE_FWUPDOFFLINE
```

Comment:
Maybe we should change the default in packaging we have in tree for non-x86 too. Offline is only used for EC on an x86 design right?

Reply:
Kinda, it's not exactly designed for x86 -- but we know on some platforms (that happen to be non-x86) that we're not going to use it.